packages/web/src/components/ai-attribution-modal/SearchBarResult.jsx

```diff
@@ -116,12 +116,12 @@ const SearchBarResult = memo((props) => {
       {!allowAiAttribution ? (
         <>
           <Tag>{messages.disabledTag}</Tag>
-          <TwitterShareButton
+          <XShareButton
             className={styles.twitterButton}
```

Comment:
Can we drop all these custom class names?

Reply:
good call